src/types.ts

```diff
@@ -108,14 +114,7 @@ declare namespace Clova {
 
   export type Card = {};
 
-  export type Directive = {
-    header: {
-      messageId: string;
-      name: string;
-      namespace: string;
-    };
-    payload: object;
-  };
+  export type Directive = audioPlayer.PlayDirective;
```

Comment:
Make `Directive = audioPlayer.PlayDirective` may work for this pull request, but we also lose the other opportunity to use other types of directive. IMO we should be better extend it.

Looking at the document [1] you provided, there are lots of different directives like:
- `AudioPlayer.Play`
- `AudioPlayer.StreamDeliver`
- `PlaybackController.Pause`
- `PlaybackController.Resume`
- `PlaybackController.Stop`

So simply make it equals to `PlayDirective` doesn't make sense to me.

[1] https://clova-developers.line.me/guide/#/CEK/References/CEK_API.md#message-fields-2

Reply:
Thank you for your reply.
That is just as you pointed out.

I added another directives.

* AudioPlayer.StreamDeliver
* PlaybackController.Pause
* PlaybackController.Resume
* PlaybackController.Stop

Is it necessary to add "{}" to "Directive" type like "OutputSpeech" type?

I'm sorry if there are mistakes.